client/my-sites/checkout/upsell-nudge/professional-email-upsell/style.scss

```
@@ -15,12 +15,80 @@
 
 .professional-email-upsell__header {
 	text-align: center;
+
+	.billing-interval-toggle {
+		display: flex;
+
+		.segmented-control {
+			background-color: #f2f2f2;
```

I don't like the use of a colour that's not in our standard palette. I think we should use `var( --studio-gray-0 )` here -- the difference is not visible to me!
```suggestion
			background-color: var( --studio-gray-0 );
```